src/components/CommentList/CommentList.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Comment } from '../Comment/Comment';
+
+export const CommentList = ({ comments }) => (
+  comments.map(comment => <Comment key={comment.id} {...comment} />)
```

Comment:
давай писати так
```suggestion
  comments.map(comment => <Comment key={comment.id} comment={comment} />)
```
Так зрозуміліше що передається